docs/source/installing.rst

```diff
@@ -6,13 +6,85 @@ root directory. Developers can run ``make install-local`` to install the dev and
 requirements alongside the base requirements. If you want a minimal installation without any
 parser specific dependencies, run ``make install-base``.
 
+
+Installation with ``conda`` on Windows
+--------------------------------------
+
+You can install and run ``unstructured`` on Windows with ``conda``, but the process
+involves a few extra steps. This section covers will help you get up and running.
+
+* Install `Anaconda <https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html>`_ on your Windows machine.
+* Install Microsoft C++ Build Tools using the instructions in `this Stackoverflow post <https://stackoverflow.com/questions/64261546/how-to-solve-error-microsoft-visual-c-14-0-or-greater-is-required-when-inst>`_. C++ build tools are required for the ``pycocotools`` dependency.
+* Run ``conda env create -f environment.yml`` using the ``environment.yml`` file in the ``unstructured`` repo to create a virtual environment. The environment will be named ``unstructured``.
+* Run ``conda activate unstructured`` to activate the virtualenvironment.
+* Run ``pip install unstructured`` to install the ``unstructured`` library.
+
+===============================================
+Setting up ``unstructured`` for local inference
+===============================================
+
+If you need to run model inferences locally, there are a few additional steps you need to
+take. The main challenge is installing ``detectron2`` for PDF layout parsing. ``detectron2``
+does not officially support Windows, but it is possible to get it install on Windows.
+The installation instructions are based on the instructions LayoutParser provides
+`here <https://layout-parser.github.io/tutorials/installation#for-windows-users>`_.
+
+* Run ``pip install pycocotools-windows`` to install a Windows compatible version of ``pycocotools``. Alternatively, you can run ``pip3 install "git+https://github.com/philferriere/cocoapi.git#egg=pycocotools&subdirectory=PythonAPI"`` as outlined in `this GitHub issue <https://github.com/cocodataset/cocoapi/issues/169#issuecomment-462528628>`_.
+* Run ``git clone https://github.com/ivanpp/detectron2.git && cd detectron2 && pip install -e .`` to install a Windows compatible version of the ``detectron2`` library.
```

Comment:
Stringing together commands via `&&` works in *sh and seems to work in a windows command prompt, but doesn't work in PowerShell. We might want to specify the terminal type.

Reply:
Added a comment on the terminal type